swarm/storage/ldbstore.go

```diff
@@ -47,15 +48,8 @@ var (
 )
 
 const (
-	defaultDbCapacity = 5000000
```

Comment:
Looks like we had two params for the LDBCapacity, but this one is 100x the `defaultLDBCapacity`. I think this it too much, because 5000000 * 4kb == 20gb.

For now I leave `defaultLDBCapacity` to be 50000*4kb == 200mb, but probably we should increase it by 10x, as a default value.

Reply:
20g is fine